fishtest/fishtest/views.py

```diff
@@ -925,26 +1014,32 @@ def tests_view(request):
     last_updated = task.get('last_updated', datetime.datetime.min)
     task['last_updated'] = last_updated
 
-  return { 'run': run, 'run_args': run_args, 'chi2': calculate_residuals(run),
+  return {'run': run, 'run_args': run_args, 'chi2': calculate_residuals(run),
           'approver': has_permission('approve_run', request.context, request),
-          'totals': '(%s active worker%s with %s core%s)' % (active, ('s' if active != 1 else ''),
-                                                             cores, ('s' if cores != 1 else ''))}
+          'totals': '(%s active worker%s with %s core%s)'
+          % (active, ('s' if active != 1 else ''),
+             cores, ('s' if cores != 1 else ''))}
+
 
 def post_result(run):
   title = run['args']['new_tag'][:23]
 
   if 'username' in run['args']:
     title += '  (' + run['args']['username'] + ')'
 
-  body = 'http://tests.stockfishchess.org/tests/view/%s\n\n' % (str(run['_id']))
+  body = 'http://tests.stockfishchess.org/tests/view/%s\n\n' % (
```

Comment:
IMO better to haven't the URL written in the body of a function (no idea if we should update also to https)

Reply:
done